Vagrantfile

```diff
@@ -1,10 +1,10 @@
 Vagrant.configure(2) do |config|
 
-    config.vm.box = "ubuntu/bionic64"
+    config.vm.box = "ubuntu/focal64"
```

Comment:
We don't actually need this defined twice in the vagrantfile (i.e. lines 3 and 7)

Reply:
This duplication is present a lot of our vagrantfiles I believe, might as well start stripping it out somewhere though! :)

Reply:
I once saw a bug if this wasn't done. Can't remember what it was and not sure if it still applies, I've just been cargo culting it since :-)

Reply:
something about destroy not working, oddly, I think?

Reply:
Fair enough! I've never encountered any problems like that, but that's not to say that don't exist in some particular set of circumstances. I also don't think it will hurt to leave it in either, just an extra line to parse